examples/ci/app/ci/types.ts

```diff
@@ -25,6 +25,12 @@ export type TestConfig<TPayload = unknown> = {
    * expected result in the Redis
    */
   expectedResult: string
+  /**
+   * whether the workflow should start
+   * 
+   * @default true
+   */
+  workflowStarts?: boolean
```

Comment:
`shouldWorkflowStart` is a more clear boolean naming, `workflowStarts` sounds like a date to me